tracing-actix-web-mozlog/src/subscriber.rs

```diff
@@ -129,7 +129,7 @@ where
             values.insert("spans".to_string(), spans.into());
 
             let v = MozLogMessage {
-                timestamp: chrono::Utc::now().timestamp_nanos(),
+                timestamp: time::OffsetDateTime::now_utc().unix_timestamp_nanos(),
```

Comment:
Let's go even further: `std::time::SystemTime` can provide this (albeit with a clunkier interface) so we could avoid the `time` crate's dep as well.

You could cast this down to 64 bits too (its max duration is something like hundreds of years) but that's not terribly important.

```suggestion
                timestamp: SystemTime::now()
                    .duration_since(SystemTime::UNIX_EPOCH)
                    .unwrap_or_default()
                    .as_nanos() as u64,
```

Reply:
Made a small tweak. Cast as i64 to keep this PR tight.